src/ui/compilationStatusView.ts

```diff
@@ -18,6 +18,10 @@ function toStatusMessage(status: CompilationStatus): string {
     return Messages.CompilationStatus.CompilationSucceeded;
   case CompilationStatus.VerificationStarted:
     return Messages.CompilationStatus.Verifying;
+  case CompilationStatus.VerificationInProgress:
```

Comment:
Would you mind formatting it like the following and use string interpolation?
```ts
    return message != null
      ? `${Messages.CompilationStatus.VerificationInProgressHeader} ${message}`
      : Messages.CompilationStatus.Verifying;
```

You could also add the corresponding eslint rule to `.eslintrc.json`:
```json
"operator-linebreak": [ "warn", "before" ]
```

Reply:
Done, thanks!